build/build_parallel/build_parallel.js

```diff
@@ -203,7 +203,7 @@ var makeTaskList = promisify(function (callback) {
 var makeTaskFunction = function (subtasks) {
   var options = {
     parallel: true,
-    continueOnError: !!config.task.continueOnError,
+    continueOnError: true,
```

Comment:
>true [](start = 21, length = 4)

commenting here just to double check that this is an intended change (since you seem to care about the value below)

Reply:
Yes,  This just makes tracking down errors easier.  If it's building 4 things in parallel and 1 fails, it finishes the other 3.  Then the summary only lists the single failure.  If this gets set to false, then it cancels the other 3 and they're all listed as failed.  (Jenkins does the same thing on failure and it makes tracking down failures really annoying.

---
In reply to: [154873753](https://github.com/Azure/azure-iot-sdk-node/pull/169#discussion_r154873753) [](ancestors = 154873753)

Reply:
The change below affects the next set of parallel tasks.  If config.task.continueOnError is set, it will continue with the next set of parallel tasks on failure.

---
In reply to: [155023758](https://github.com/Azure/azure-iot-sdk-node/pull/169#discussion_r155023758) [](ancestors = 155023758,154873753)